tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -0,0 +1,347 @@
+        # response = self.client.get(links['previous']['href'], format='json')
+        # assert response.status_code == 200
+        # assert len(response.data) == 0
+
+        # group3 = self.create_group(
+        #     checksum='c' * 32,
+        #     last_seen=now + timedelta(seconds=1),
+        # )
+
+        # links = self._parse_links(response['Link'])
+
+        # assert links['previous']['results'] == 'false'
+        # assert links['next']['results'] == 'true'
+
+        # print(links['previous']['cursor'])
+        # response = self.client.get(links['previous']['href'], format='json')
+        # assert response.status_code == 200
+        # assert len(response.data) == 1
+        # assert response.data[0]['id'] == six.text_type(group3.id)
+
+    def test_stats_period(self):
+        # TODO(dcramer): this test really only checks if validation happens
+        # on statsPeriod
+        now = timezone.now()
+        self.create_group(
+            checksum='a' * 32,
+            last_seen=now - timedelta(seconds=1),
+        )
+        self.create_group(
+            checksum='b' * 32,
+            last_seen=now,
+        )
+
+        self.login_as(user=self.user)
+
+        response = self.client.get(u'{}?statsPeriod=24h'.format(self.path), format='json')
+        assert response.status_code == 200
+
+        response = self.client.get(u'{}?statsPeriod=14d'.format(self.path), format='json')
+        assert response.status_code == 200
+
+        response = self.client.get(u'{}?statsPeriod='.format(self.path), format='json')
+        assert response.status_code == 200
+
+        response = self.client.get(u'{}?statsPeriod=48h'.format(self.path), format='json')
+        assert response.status_code == 400
+
+    def test_environment(self):
+        self.create_environment(name='production', organization=self.project.organization)
+        self.create_event(tags={'environment': 'production'})
+
+        self.login_as(user=self.user)
+
+        response = self.client.get(self.path + '?environment=production', format='json')
+        assert response.status_code == 200
+
+        response = self.client.get(self.path + '?environment=garbage', format='json')
+        assert response.status_code == 404
+
+    def test_auto_resolved(self):
+        project = self.project
+        project.update_option('sentry:resolve_age', 1)
+        now = timezone.now()
+        self.create_group(
+            checksum='a' * 32,
+            last_seen=now - timedelta(days=1),
+        )
+        group2 = self.create_group(
+            checksum='b' * 32,
+            last_seen=now,
+        )
+
+        self.login_as(user=self.user)
+        response = self.client.get(self.path, format='json')
+        assert response.status_code == 200
+        assert len(response.data) == 1
+        assert response.data[0]['id'] == six.text_type(group2.id)
+
+    # TODO(jess): figure out how these all should work with multi project
+
+    # def test_lookup_by_event_id(self):
```

Comment:
Should you also have tests ensuring organization/project permissions are enforced?